### PR TITLE
docs: [DHIS2-11200][DHIS2-11207] enrollment view/edit event page

### PR DESCRIPTION
**Pull request by simonadomnisoru**

### CODE REVIEW

File: user/using-the-capture-app.md [hunk 1]
```diff
@@ -872,4 +872,10 @@ On the enrollment dashboard, you can view the Person Profile widget.
 
 The Person profile widget is used for viewing key attributes but not for editing. In order to edit the person profile, you must select the `Edit` button to open an edit window. 
 
+## Enrollment event view and edit page
```
Comment (larshelge) [inline user/using-the-capture-app.md:876]: Do we want to add some text to this section?

Comment (simonadomnisoru) [reply user/using-the-capture-app.md:876]: Hey @larshelge, 

The text is added bellow under `Reaching the enrollment event via url`, do you think I should add extra text? Thanks!

Comment (simonadomnisoru) [reply user/using-the-capture-app.md:876]: I added the text from the section below. I hope it is better now. Thanks!

**Review by Philip-Larsen-Donnelly — APPROVED** [review]
Seems in line with current state of development.

**Pull request merged**